### PR TITLE
Bug 2041750: [Alibaba] fix endpoint error in some regions

### PR DESCRIPTION
**Pull request by bd233**
Update sdk and terraform provier version, and add some endpoints of ECS
service to fix endpoint error.

### CODE REVIEW

**Comment by patrickdillon** [comment]
Can you separate the vendor code into a separate commit so this is easier to review?

**Comment by bd233** [comment]
> Can you separate the vendor code into a separate commit so this is easier to review?

Have done.

**Comment by kirankt** [comment]
The golint job failure is real. Please fix. Always run the checks listed in CONTRIBUTING.md file before your commits to catch these types of failures.

File: pkg/asset/installconfig/alibabacloud/client.go [hunk 1]
```diff
@@ -331,6 +323,23 @@ func (client *Client) GetAvailableZonesByInstanceType(instanceType string) ([]st
 	return zones, nil
 }
 
+func getEndpoint(productName string, regionId string) string {
```
Comment (patrickdillon) [inline pkg/asset/installconfig/alibabacloud/client.go:326]: golint is failing because `regionId` should be `regionID`

**Review by patrickdillon — COMMENTED** [review]
/approve

This looks good but you need to fix the issue I pointed out with the golint to pass the required tests.

File: pkg/asset/installconfig/alibabacloud/client.go (hunk 1 above)
Comment (bd233) [reply pkg/asset/installconfig/alibabacloud/client.go]: Have done

**Review by kirankt — APPROVED** [review]
/lgtm

**Pull request merged**